src/modules/zendesk/controllers/zendesk.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Body, Controller, Post, UseGuards } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { CreateTicketRequestDto } from '../dto/zendesk.dto';
+import { ZendeskService } from '../services/zendesk.service';
+import { CreateTicketResponse } from '../zendesk.types';
+
+import { Permission } from '~modules/users/types';
+import { RequireAnyPermissions } from '~shared/decorators/require-any-permissions.decorator';
+import { LoggedInGuard } from '~shared/guards/logged-in.guard';
+
+@UseGuards(LoggedInGuard)
+@ApiTags('Zendesk')
+@Controller('zendesk')
+export class ZendeskController {
+	@Post('/support')
+	@ApiOperation({ description: 'Create ticket through Zendesk API' })
+	@RequireAnyPermissions(Permission.CREATE_TICKET_ZENDESK)
```

Comment:
Anonimous users can also report issues, so we should not require any permissions
We might want to limit the rate of requests to this endpoint though. eg: 1 per minute
You can use [nestjs/throttler](https://github.com/nestjs/throttler)
```suggestion
```